docs/Metrics-System.md

```diff
@@ -51,20 +51,36 @@ The following shows the details of the available metrics.
 * CapacityTotal: Total capacity of the file system in bytes.
 * CapacityUsed: Used capacity of the file system in bytes.
 * CapacityFree: Free capacity of the file system in bytes.
+* PathsTotal: Total number of files and directories in the file system.
 * UnderFsCapacityTotal: Total capacity of the under file system in bytes.
 * UnderFsCapacityUsed: Used capacity of the under file system in bytes.
 * UnderFsCapacityFree: Free capacity of the under file system in bytes.
 * Workers: Number of the workers.
-* FilesTotal: Total number of files and directories in the file system.
-* FilesCheckpointed: Total number of the files checkpointed.
-* FilesCreated: Total number of files and directories created.
-* FilesDeleted: Total number of files and directories deleted.
+* DirectoriesCreated: Total number of directories created.
```

Comment:
Ok. 
Shall I  use the names you provided me some comments ago ?
(i.e.
* General
* Logical Operations
* RPC Invocations
)

I was thinking about providing a brief description of this classification in the section above metrics list.
It could be something like:

-----------------

# Supported Metrics

Metrics are classified as:

* General: overall measures of the cluster (e.g. CapacityTotal).
* Logical Operations: number of operations performed (e.g. FilesCreated).
* RPC Invocations: number of RPC invocations per operation (e.g. CreateFileOps).

The following shows the details of the available metrics.

---------------

What do you think?